src/scene/components/camera_data.rs

```diff
@@ -1,13 +1,37 @@
 use nalgebra_glm::{Mat4, Vec3};
 
+enum ProjectionData {
+    Perspective {
+        fov: f32,
+        z_near: f32,
+        z_far: f32,
+    },
+    Orthographic {
+        world_height: f32,
+        z_near: f32,
+        z_far: f32,
+    },
+}
+
+impl ProjectionData {
+    /// get_projection calculates a new projection for the specified viewport width & height. TODO: Div by 0 possible for orthographic.
+    fn get_projection(&self, width: f32, height: f32) -> Mat4{
+        match self{
+            ProjectionData::Perspective { fov, z_near, z_far } => 
+                nalgebra_glm::perspective_fov_lh_no(*fov, width, height, *z_near, *z_far),
```

Comment:
Let's use RH here since GLTF already uses that. Optionally we can change this in a different PR as it might breaking rendering if we change it in this branch.

Reply:
We can merge this now and change it when pbr with the changes is ready?